### PR TITLE
feat: edit minor things

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [x] Not that difficult

### Type of Change
- [x] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [x] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
Please review the **file changes** instead of the commits!

### CODE REVIEW

File: apps/web/src/services/dashboards/dashboard-customize/modules/DashboardDefaultWidgetTab.vue [hunk 1]
```diff
@@ -87,6 +87,7 @@ export default {
         /* Event */
         const handleChangeTab = (selectedTab) => {
             tabState.activeTab = selectedTab;
+            state.selectedWidgetConfigId = undefined;
```
Comment (yuda110) [inline apps/web/src/services/dashboards/dashboard-customize/modules/DashboardDefaultWidgetTab.vue:90]: Init selected widget(red box) when change tab.

스크린샷 2023-06-01 오후 1 45 57

File: apps/web/src/services/dashboards/dashboard-customize/modules/dashboard-widget-input-form/DashboardWidgetInputForm.vue [hunk 2]
```diff
@@ -37,7 +37,7 @@
                               :class="{inherit: inheritableProperties.includes(propertyName)}"
                         >{{ $t('DASHBOARDS.CUSTOMIZE.ADD_WIDGET.INHERIT') }}</span>
                         <p-toggle-button :value="inheritableProperties.includes(propertyName)"
-                                         :disabled="widgetOptionsJsonSchema.properties?.[propertyName]?.disabled"
+                                         :disabled="widgetOptionsJsonSchema.properties?.[propertyName]?.disabled || fixedProperties.includes(propertyName)"
```
Comment (yuda110) [inline apps/web/src/services/dashboards/dashboard-customize/modules/dashboard-widget-input-form/DashboardWidgetInputForm.vue:40]: description: disable toggle button when it's fixed options(ex. group by)

스크린샷 2023-06-01 오후 1 47 10

File: apps/web/src/services/dashboards/dashboard-customize/modules/DashboardDefaultWidgetTab.vue (hunk 1 above)
Comment (yuda110) [reply apps/web/src/services/dashboards/dashboard-customize/modules/DashboardDefaultWidgetTab.vue:90]: After changing tab
스크린샷 2023-06-01 오후 1 55 50

File: apps/web/src/services/dashboards/widgets/severity-status-by-service/SeverityStatusByServiceWidget.vue [hunk 3]
```diff
@@ -16,7 +16,7 @@
                 </div>
             </div>
             <div class="legend-wrapper">
-                <div v-for="status in Object.values(SEVERITY_STATUS)"
+                <div v-for="status in Object.values(SEVERITY_STATUS_MAP)"
```
Comment (WANZARGEN) [inline apps/web/src/services/dashboards/widgets/severity-status-by-service/SeverityStatusByServiceWidget.vue:19]: nitpick: This is good but it executes `Object.values(SEVERITY_STATUS_MAP) `everytime.. for more efficient way, you can change it to use just a constant value. :)

**Review by WANZARGEN — APPROVED** [review]
Great Overally~ There seemed to be a critical issue, so I left a comment.

File: apps/web/src/services/dashboards/widgets/_components/WidgetDataTable.vue [hunk 4]
```diff
@@ -159,16 +159,14 @@
             >
                 <template #default>
                     <span class="this-page">{{ state.proxyThisPage }}</span>
-                    <span v-if="showNextPage"> / ...</span>
+                    <span v-if="props.showNextPage"> / ...</span>
```
[the remark this answers is not in the document]
Comment (yuda110) [reply apps/web/src/services/dashboards/widgets/_components/WidgetDataTable.vue:162]: However, it works fine even without it. It's strange...☹️

[the remark this answers is not in the document]
Comment (piggggggggy) [reply apps/web/src/services/dashboards/widgets/_components/WidgetDataTable.vue:162]: Question: doesn't it matter whether to use `props.` or not?

File: apps/web/src/services/dashboards/widgets/total-failure-and-severity/TotalFailureAndSeverityWidget.vue [hunk 5]
```diff
@@ -120,6 +120,9 @@ import { useWidgetState } from '@/services/dashboards/widgets/_hooks/use-widget-
 import type { XYChartData } from '@/services/dashboards/widgets/type';
 
 
+// type Data = CostAnalyzeDataModel['results'];
+type Data = any[];
+
```
Comment (piggggggggy) [inline apps/web/src/services/dashboards/widgets/total-failure-and-severity/TotalFailureAndSeverityWidget.vue:125]: Question: to be Improved?

**Review by sulmoJ — APPROVED** [review]
LGTM!

**Review by piggggggggy — APPROVED** [review]
LGTM! I left some question (not important)

File: apps/web/src/services/dashboards/widgets/total-failure-and-severity/TotalFailureAndSeverityWidget.vue (hunk 5 above)
Comment (yuda110) [reply apps/web/src/services/dashboards/widgets/total-failure-and-severity/TotalFailureAndSeverityWidget.vue:125]: yeap!

**Pull request merged**